### ISSUE TITLE
20210827 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#2423]
# 20210827 Github 前端趋势榜
Github 前端开发日报栏目数据来自[Github热榜](http://news.caibaojian.com.cn/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [microsoft/Web-Dev-For-Beginners](https://github.com/microsoft/Web-Dev-For-Beginners) （24 Lessons, 12 Weeks, Get Started as a Web Developer）
* [google/zx](https://github.com/google/zx) （A tool for writing better scripts）
* [louislam/uptime-kuma](https://github.com/louislam/uptime-kuma) （A fancy self-hosted monitoring tool）
* [careercup/CtCI-6th-Edition-JavaScript](https://github.com/careercup/CtCI-6th-Edition-JavaScript) （Cracking the Coding Interview 6th Ed. JavaScript Solutions）
* [ryanburgess/engineer-manager](https://github.com/ryanburgess/engineer-manager) （A list of engineering manager resource links.）
* [goldbergyoni/nodebestpractices](https://github.com/goldbergyoni/nodebestpractices) （&#x2705; The Node.js best practices list (June 2020)）
* [vuejs/vue](https://github.com/vuejs/vue) （        A progressive, incrementally-adoptable JavaScript framework for building UI on the web.      ）
* [mui-org/material-ui](https://github.com/mui-org/material-ui) （React components that implement Google's Material Design.）
* [chokcoco/CSS-Inspiration](https://github.com/chokcoco/CSS-Inspiration) （CSS Inspiration，在这里找到写 CSS 的灵感！）
* [MoonBegonia/ninja](https://github.com/MoonBegonia/ninja) （）
* [airbnb/javascript](https://github.com/airbnb/javascript) （        JavaScript Style Guide      ）
* [qier222/YesPlayMusic](https://github.com/qier222/YesPlayMusic) （高颜值的第三方网易云播放器，支持 Windows / macOS / Linux）
* [sudheerj/reactjs-interview-questions](https://github.com/sudheerj/reactjs-interview-questions) （List of top 500 ReactJS Interview Questions &amp; Answers....Coding exercise questions are coming soon!!）
* [microsoft/playwright](https://github.com/microsoft/playwright) （Node library to automate Chromium, Firefox and WebKit with a single API）
* [LeCoupa/awesome-cheatsheets](https://github.com/LeCoupa/awesome-cheatsheets) （? Awesome cheatsheets for popular programming languages, frameworks and development tools. They include everything you should know in one single file.）
* [Microsoft/monaco-editor](https://github.com/Microsoft/monaco-editor) （A browser based code editor）
* [YMFE/yapi](https://github.com/YMFE/yapi) （YApi 是一个可本地部署的、打通前后端及QA的、可视化的接口管理平台）
* [quasarframework/quasar](https://github.com/quasarframework/quasar) （        Quasar Framework      ）
* [ascoders/weekly](https://github.com/ascoders/weekly) （前端精读周刊。帮你理解最前沿、实用的技术。）
* [ethereumbook/ethereumbook](https://github.com/ethereumbook/ethereumbook) （The Ethereum Book Project）
* [jojoldu/junior-recruit-scheduler](https://github.com/jojoldu/junior-recruit-scheduler) （        주니어 개발자 채용 정보      ）
* [SudhanPlayz/Discord-MusicBot](https://github.com/SudhanPlayz/Discord-MusicBot) （An advanced discord music bot, supports Spotify, Soundcloud, YouTube with Shuffling, Volume Control and Web Dashboard with Slash Commands support!）
* [alshedivat/al-folio](https://github.com/alshedivat/al-folio) （A beautiful, simple, clean, and responsive Jekyll theme for academics）
* [kenwheeler/slick](https://github.com/kenwheeler/slick) （the last carousel you'll ever need）
* [Collective-SB/ISAN](https://github.com/Collective-SB/ISAN) （A multilateration solution from the Collective, written in YOLOL, for use in Starbase.）

## Vue
* [PanJiaChen/vue-element-admin](https://github.com/PanJiaChen/vue-element-admin) （        vue2 admin / a management system template      ）
* [abhisheknaiidu/awesome-github-profile-readme](https://github.com/abhisheknaiidu/awesome-github-profile-readme) （&#x1f60e; A curated list of awesome Github Profile READMEs &#x1f4dd;）
* [element-plus/element-plus](https://github.com/element-plus/element-plus) （&#x1f389; A Vue.js 3.0 UI Library）
* [anncwb/vue-vben-admin](https://github.com/anncwb/vue-vben-admin) （&#x2728; &#x2728; &#x2728; A vue3 style Admin based on Vite vue3.0, vite, ant design vue 2.x, typescript，Efforts to update in progress...）
* [bastienwirtz/homer](https://github.com/bastienwirtz/homer) （A very simple static homepage for your server.）
* [chuzhixin/vue-admin-beautiful-pro](https://github.com/chuzhixin/vue-admin-beautiful-pro) （&#x1f680;&#x1f680;&#x1f680;vue3,vue3.0,vue,vue3.x,vue.js,vue后台管理,admin,vue-admin,vue-element-admin,ant-design，vue-admin-beautiful-pro,vab admin pro,vab admin plus主线版本基于element-plus、element-ui、ant-design-vue三者并行开发维护，同时支持电脑，手机，平板，切换分支查看不同的vue版本，element-plus版本已发布(vue3,vue3.0,vue,vue3.x,vue.js)程序无国界，但程序员有国界，中国国家尊严不容挑衅，如果您在特殊时期继续购买HM、耐克、阿迪达斯等品牌那么您将无权继续使用Vab）
* [uasoft-indonesia/badaso](https://github.com/uasoft-indonesia/badaso) （The First Open-Source Laravel Vue headless CMS (PWA dashboard + CRUD + API generator &amp; more) for more productivity !）
* [shfshanyue/Daily-Question](https://github.com/shfshanyue/Daily-Question) （每天一道面试题，有关前端，后端，devops以及软技能，促进职业成长，敲开大厂之门。）
* [JakHuang/form-generator](https://github.com/JakHuang/form-generator) （&#x2728;Element UI表单设计及代码生成器）
* [lin-xin/vue-manage-system](https://github.com/lin-xin/vue-manage-system) （        The web management system solution based on Vue2 and Element-UI.      ）
* [vueComponent/ant-design-vue](https://github.com/vueComponent/ant-design-vue) （An enterprise-class UI components based on Ant Design and Vue. ?）
* [bailicangdu/vue2-manage](https://github.com/bailicangdu/vue2-manage) （        基于 vue + element-ui 的后台管理系统      ）
* [chaitin/xray](https://github.com/chaitin/xray) （一款完善的安全评估工具，支持常见 web 安全问题扫描和自定义 poc | 使用之前务必先阅读文档）
* [CareyWang/sub-web](https://github.com/CareyWang/sub-web) （）
* [macrozheng/mall-admin-web](https://github.com/macrozheng/mall-admin-web) （mall-admin-web是一个电商后台管理系统的前端项目，基于Vue+Element实现。 主要包括商品管理、订单管理、会员管理、促销管理、运营管理、内容管理、统计报表、财务管理、权限管理、设置等功能。）
* [newbee-ltd/vue3-admin](https://github.com/newbee-ltd/vue3-admin) （Vue 3.0 + Vite 2.0 + Vue-Router 4.0 + Element-Plus + Echarts 5.0 + Axios 开发的后台管理系统）
* [xiaoxian521/vue-pure-admin](https://github.com/xiaoxian521/vue-pure-admin) （&#x2728; &#x1f680;Vue3.0+TypeScript+Vite2.0+Element-Plus编写的一套后台管理系统（兼容移动端））
* [hay-kot/mealie](https://github.com/hay-kot/mealie) （）
* [CryptoBlades/cryptoblades](https://github.com/CryptoBlades/cryptoblades) （）
* [pipipi-pikachu/PPTist](https://github.com/pipipi-pikachu/PPTist) （基于 Vue3.x + TypeScript 的在线演示文稿应用）
* [jsbroks/coco-annotator](https://github.com/jsbroks/coco-annotator) （&#x270f;&#xfe0f; Web-based image segmentation tool for object detection and localization）
* [hinesboy/mavonEditor](https://github.com/hinesboy/mavonEditor) （mavonEditor - A markdown editor based on Vue that supports a variety of personalized features）
* [Geek-James/ddBuy](https://github.com/Geek-James/ddBuy) （&#x1f389;Vue全家桶+Vant 搭建大型单页面电商项目.http://ddbuy.7-orange.cn）
* [rowanwins/vue-dropzone](https://github.com/rowanwins/vue-dropzone) （A Vue.js component for Dropzone.js - a drag’n’drop file uploads utility with image previews）
* [vuejs/docs](https://github.com/vuejs/docs) （Vue 3 core documentation）

## HTML
* [mattermost/focalboard](https://github.com/mattermost/focalboard) （Focalboard is an open source, self-hosted alternative to Trello, Notion, and Asana.）
* [gustavoguanabara/html-css](https://github.com/gustavoguanabara/html-css) （Curso de HTML5 e CSS3）
* [google/styleguide](https://github.com/google/styleguide) （        Style guides for Google-originated open-source projects      ）
* [projectcalico/calico](https://github.com/projectcalico/calico) （Cloud native networking and network security）
* [vector-im/element-web](https://github.com/vector-im/element-web) （A glossy Matrix collaboration client for the web.）
* [freddier/hyperblog](https://github.com/freddier/hyperblog) （Un blog increíble para el curso de Git y Github de Platzi）
* [facebookresearch/fastText](https://github.com/facebookresearch/fastText) （Library for fast text representation and classification.）
* [PeiQi0/PeiQi-WIKI-POC](https://github.com/PeiQi0/PeiQi-WIKI-POC) （鹿不在侧，鲸不予游&#x1f40b;）
* [cod3rcursos/web-moderno](https://github.com/cod3rcursos/web-moderno) （）
* [mdn/learning-area](https://github.com/mdn/learning-area) （Github repo for the MDN Learning Area.）
* [Flam3rboy/discord-bot-client](https://github.com/Flam3rboy/discord-bot-client) （A custom version of discord, with bot login support）
* [foundation/foundation-sites](https://github.com/foundation/foundation-sites) （The most advanced responsive front-end framework in the world. Quickly create prototypes and production code for sites that work on any kind of device.）
* [unclestrong/DeepLearning_LHY21_Notes](https://github.com/unclestrong/DeepLearning_LHY21_Notes) （深度学习 李宏毅 2021 学习笔记）
* [NorthwoodsSoftware/GoJS](https://github.com/NorthwoodsSoftware/GoJS) （JavaScript diagramming library for interactive flowcharts, org charts, design tools, planning tools, visual languages.）
* [opensaludlab/ciencia_datos](https://github.com/opensaludlab/ciencia_datos) （Repositorio del programa de formación en ciencia de datos de la Academia de OpenSalud LAB）
* [desandro/masonry](https://github.com/desandro/masonry) （        ? Cascading grid layout library      ）
* [rfordatascience/tidytuesday](https://github.com/rfordatascience/tidytuesday) （Official repo for the #tidytuesday project）
* [OWASP/railsgoat](https://github.com/OWASP/railsgoat) （A vulnerable version of Rails that follows the OWASP Top 10）
* [Syknapse/Contribute-To-This-Project](https://github.com/Syknapse/Contribute-To-This-Project) （This project is waiting for your contribution. If you have never contributed code on GitHub before, this is the project to get you started.）
* [tencentyun/qcloud-documents](https://github.com/tencentyun/qcloud-documents) （腾讯云官方文档）
* [cypress-io/cypress-example-kitchensink](https://github.com/cypress-io/cypress-example-kitchensink) （This is an example app used to showcase Cypress.io testing.）
* [goitacademy/javascript-homework](https://github.com/goitacademy/javascript-homework) （）
* [OCA/sale-workflow](https://github.com/OCA/sale-workflow) （Odoo Sales, Workflow and Organization）
* [MicrosoftDocs/msteams-docs](https://github.com/MicrosoftDocs/msteams-docs) （Source for the Microsoft Teams developer platform documentation.）
* [xfirefly/Airplay-SDK](https://github.com/xfirefly/Airplay-SDK) （Airplay Receiver SDK supports Airplay Mirroring and AirPlay Casting to a receiver device.）

## CSS
* [blueedgetechno/windows11](https://github.com/blueedgetechno/windows11) （windows 11 in react &#x1f4bb;&#x1f308;&#x26a1;）
* [bradtraversy/50projects50days](https://github.com/bradtraversy/50projects50days) （50+ mini web projects using HTML, CSS &amp; JS）
* [necolas/normalize.css](https://github.com/necolas/normalize.css) （A collection of HTML element and attribute style-normalizations      ）
* [elrumo/macOS_Big_Sur_icons_replacements](https://github.com/elrumo/macOS_Big_Sur_icons_replacements) （Replacement icons for popular apps in the style of macOS Big Sur）
* [dunovank/jupyter-themes](https://github.com/dunovank/jupyter-themes) （        Custom Jupyter Notebook Themes      ）
* [Remix-Design/remixicon](https://github.com/Remix-Design/remixicon) （Open source neutral style icon system）
* [WebStackPage/WebStackPage.github.io](https://github.com/WebStackPage/WebStackPage.github.io) （&#x2764;&#xfe0f;静态响应式网址导航网站 - webstack.cc）
* [progit/progit2](https://github.com/progit/progit2) （        Pro Git 2nd Edition      ）
* [cool-team-official/cool-admin-vue](https://github.com/cool-team-official/cool-admin-vue) （cool-admin一个很酷的后台权限管理框架，模块化、插件化、CRUD极速开发，永久开源免费，基于midway.js 2.0、typeorm、mysql、jwt、element-ui、vuex、vue-router、vue等构建）
* [liangjingkanji/DrakeTyporaTheme](https://github.com/liangjingkanji/DrakeTyporaTheme) （&#x1f359; typora theme for simple flat）
* [john-smilga/react-projects](https://github.com/john-smilga/react-projects) （）
* [primefaces/primeng](https://github.com/primefaces/primeng) （        UI Components for Angular      ）
* [jonsuh/hamburgers](https://github.com/jonsuh/hamburgers) （        Tasty CSS-animated Hamburgers      ）
* [erikflowers/weather-icons](https://github.com/erikflowers/weather-icons) （        215 Weather Themed Icons and CSS      ）
* [houshanren/hangzhou_house_knowledge](https://github.com/houshanren/hangzhou_house_knowledge) （2017年买房经历总结出来的买房购房知识分享给大家，希望对大家有所帮助。买房不易，且买且珍惜。Sharing the knowledge of buy an own house that according to the experience at hangzhou in 2017 to all the people. It's not easy to buy a own house, so I hope that it would be useful to everyone.）
* [matcornic/hugo-theme-learn](https://github.com/matcornic/hugo-theme-learn) （Porting Grav Learn theme to Hugo）
* [elsa-workflows/elsa-core](https://github.com/elsa-workflows/elsa-core) （A .NET Standard 2.0 Workflows Library）
* [Microsoft/vscode-docs](https://github.com/Microsoft/vscode-docs) （Public documentation for Visual Studio Code）
* [microsoft/SmartHotel360-Website](https://github.com/microsoft/SmartHotel360-Website) （SmartHotel360 Webiste）
* [udacity/fend](https://github.com/udacity/fend) （General Front End Nanodegree Content Resources）
* [apachecn/sklearn-doc-zh](https://github.com/apachecn/sklearn-doc-zh) （&#x1f4d6; [译] scikit-learn（sklearn） 中文文档）
* [ihollander/phase-3-mock-code-challenge-spicemaster](https://github.com/ihollander/phase-3-mock-code-challenge-spicemaster) （）
* [gatsbyjs/gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default) （The default Gatsby starter）
* [goitacademy/react-homework](https://github.com/goitacademy/react-homework) （）
* [gabrielecirulli/2048](https://github.com/gabrielecirulli/2048) （        A small clone of 1024 (https://play.google.com/store/apps/details?id=com.veewo.a1024)      ）

## 关于我
 [前端开发博客](http://caibaojian.com.cn/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](https://open.weixin.qq.com/qr/code?username=caibaojian_com)。
## 推荐
 关注我的微信公众号：[前端开发博客](https://open.weixin.qq.com/qr/code?username=caibaojian_com)，在后台回复以下关键字获取资源：
 1. 回复「1024」，领取前端进阶资料，包含小程序、Nodejs、Git等
 2. 回复「Vue」获取 Vue 精选文章
 3. 回复「面试」获取 面试 精选文章
 4. 回复「JS」获取 JavaScript 精选文章
 5. 回复「CSS」获取 CSS 精选文章
 6. 回复「加群」进入500人前端精英群
 7. 回复「电子书」下载我整理的大量前端资源，含面试、Vue实战项目、CSS和JavaScript电子书等。
 8. 回复「知识点」下载高清JavaScript知识点图谱
